frb_example/pure_dart/rust/src/api.rs

```diff
@@ -537,3 +537,15 @@ pub fn get_complex_array() -> [Point; 2] {
 pub fn get_usize(u: usize) -> usize {
     u
 }
+
+/// Example for @freezed and @meta.immutable
+#[frb(metadata=("freezed", "immutable" import "package:meta/meta.dart" as meta))]
```

Comment:
I guess this means *annotations* for dart classes? If so, what about remaming "metadata=..." to "dart_annotation=..."

Reply:
Good point! I did not realized they call the `int` in `int a = ...` a type annotation before. So `dart_metadata` is quite reasonable.